MaxText/configs/base.yml

```diff
@@ -247,3 +247,28 @@ decode_sampling_temperature: 1.
 
 eval_interval: -1  # the specific number of train step between eval_step
 target_eval_loss: 0.  # early stop once reaching target eval_loss
+
+# Vertex AI Tensorboard Configurations
+# Set to True to create Vertex Tensorboard instance in GCP
+# Set to False if running via XPK as Vertex Tensorboard and Experiment will be created by XPK
```

Comment:
Can you explain in what settings you'll want the different values here? Do we always set this to true for local runs to create the Vertex Tensorboard? Or?

Reply:
Added details about different scenarios in README.